bindings/src/Capgemini.PowerApps.SpecFlowBindings/Steps/LookupSteps.cs

```diff
@@ -124,5 +124,11 @@ public static void WhenISelectARelatedLookupInTheForm(string lookupName)
             Driver.WaitUntilAvailable(By.XPath(AppElements.Xpath[AppReference.Entity.LookupFieldExistingValue].Replace("[NAME]", lookupName))).Click();
             Driver.WaitForTransaction();
         }
+
```

Comment:
/// <summary>
        /// Open the record by clicking on the Advanced lookup link.
        /// </summary>
        /// <param name="lookupField">The name of the lookup.</param>

Reply:
Just a little thought: we could add comment to keep consistency with the rest of the steps and for clarity?

Reply:
Ah yes!